### ISSUE TITLE
Documentation references sasToken in init method; param not used.

### ISSUE THREAD

**Issue by mjyeaney** [ytechie/eventhubs-js#11]
The README file references using only the hub namespace, name, and sas token to initialize the library. However, the `sasToken` parameter doesn't seem to be used in the `init()` method (that I can find). This also results in an error from `crypto.js` resulting from the empty key.

Here is the snipped from README which causes the error upon sending:

```
eventHubs.init({
    hubNamespace: eventHubsNamespace,
    hubName: eventHubsHubName,
    sasToken: sasToken
});
```

Using the full `init` signature (shown below) works perfectly:

```
eventHubs.init({
    hubNamespace: eventHubsNamespace,
    hubName: eventHubsHubName,
    keyName: eventHubsKeyName,
    key: eventHubsKey
});
```

Not sure if this is just an issue with old documentation / etc., but figured it was worth pointing out.

**Comment by ytechie** [comment]
Good catch. Someone else's check-in broke it. I'll fix it.

**Comment by ytechie** [comment]
This should be fixed now in 1.1.3.
